DP02_04b_Intermediate_Butler_Queries.ipynb

```diff
@@ -9,8 +9,8 @@
     "<img align=\"left\" src = https://project.lsst.org/sites/default/files/Rubin-O-Logo_0.png width=250 style=\"padding: 10px\" alt=\"Rubin Observatory logo, a graphical representation of turning stars into data.\">\n",
```

Comment:
<p>I think you want to keep the <code>[0]</code> to pull out the first element, like the original had.  It stringifies a lot more legibly when it's not in a list.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">exposure:
  instrument: 'LSSTCam-imSim'
  id: 971990
  physical_filter: 'z_sim_1.4'
  obs_id: '971990'
  exposure_time: 30.0
  dark_time: 30.0
  observation_type: 'science'
  observation_reason: 'imsim'
  day_obs: 20251201
  seq_num: 0
  group_name: '971990'
  group_id: 971990
  target_name: 'UNKNOWN'
  science_program: '971990'
  tracking_ra: 70.37699524983329
  tracking_dec: -37.17573628348882
  sky_angle: 292.6518874660149
  zenith_angle: 19.469782456426444
  timespan: Timespan(begin=astropy.time.Time('2025-12-02 03:11:50.174400', scale='tai', format='iso'), end=astropy.time.Time('2025-12-02 03:12:20.174000', scale='tai', format='iso'))
</pre><p>vs</p>

<pre class="ql-syntax" spellcheck="false">[exposure.RecordClass(instrument='LSSTCam-imSim', id=971990, physical_filter='z_sim_1.4', obs_id='971990', exposure_time=30.0, dark_time=30.0, observation_type='science', observation_reason='imsim', day_obs=20251201, seq_num=0, group_name='971990', group_id=971990, target_name='UNKNOWN', science_program='971990', tracking_ra=70.37699524983329, tracking_dec=-37.17573628348882, sky_angle=292.6518874660149, zenith_angle=19.469782456426444, timespan=Timespan(begin=astropy.time.Time('2025-12-02 03:11:50.174400', scale='tai', format='iso'), end=astropy.time.Time('2025-12-02 03:12:20.174000', scale='tai', format='iso')))]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Another good tip -- it does look a lot better when you extract the list item.</p>